Source/DafnyLanguageServer.Test/DafnyLanguageServerTestBase.cs

```diff
@@ -43,16 +55,60 @@ public class DafnyLanguageServerTestBase : LanguageServerTestBase {
 
     public DafnyLanguageServerTestBase() : base(new JsonRpcTestOptions(LoggerFactory.Create(builder => builder.AddConsole().SetMinimumLevel(LogLevel.Warning)))) { }
 
-    protected override (Stream clientOutput, Stream serverInput) SetupServer() {
+
+
+    protected virtual async Task<ILanguageClient> InitializeClient(
+      Action<LanguageClientOptions> clientOptionsAction = null,
+      [CanBeNull] Action<LanguageServerOptions> serverOptionsAction = null) {
+      var client = CreateClient(clientOptionsAction, serverOptionsAction);
+      await client.Initialize(CancellationToken).ConfigureAwait(false);
```

Comment:
Why do you need the configureawait(false) here?

Reply:
I don't know. I copied this from `OmniSharp.Extensions.LanguageProtocol.Testing.LanguageServerTestBase`